.github/workflows/push.yml

```diff
@@ -0,0 +1,38 @@
+name: Build
+on:
+  push:
+    branches:
+      - master
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Install Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16.x
+
+      - name: Install Hugo
+        uses: peaceiris/actions-hugo@v2
+        with:
+          hugo-version: '0.82.0'
+          extended: true
+
+      - name: Check out branch
+        uses: actions/checkout@v2
+
+      - name: Fetch the latest pulumi/theme
+        run: |
+          hugo mod get github.com/pulumi/theme
```

Comment:
Should we consider running `hugo mod tidy` as well?

Reply:
@praneetloke looks like this might not work. I opened this PR which happens to use a different branch of `theme`, and I'm seeing conflicts: https://github.com/pulumi/pulumi-hugo/pull/777

Reply:
I see we've taken some PRs since this was merged without issue, so I'm going to call this okay for now. 🤞  Thanks for the nudge on it!

Reply:
Sounds good, Christian! I totally missed the GH notifications until now and funny to see the progress in the comments 😄